docker/1.13/Dockerfile.cpu

```diff
@@ -41,7 +41,7 @@ ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1
 
 # cython, falcon, gunicorn
 RUN \
-    ${PIP} install cython falcon gunicorn gevent requests
+    ${PIP} install cython==0.29.10 falcon==2.0.0 gunicorn==19.9.0 gevent==1.4.0 requests==2.22.0
```

Comment:
- why only for 1.13?
- why divergent pip args for cpu and gpu?
- what about ei?

Reply:
Pinning version is a requirement for the asimov containers. only 1.13 tfs containers are merged with the asimov dependencies.  The discrepancies of dependencies are from the asimov side.

Reply:
The requests version for the rest of the Python3 Docker files are pinned to 2.21.0. Are we sure we want to pin the version here to be 2.22.0?